pkg/scheduler/scheduler_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"k8s.io/utils/ptr"
```

Comment:
This import line needs to be put into the next import block.

Reply:
done.